CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.33](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.32...v4.0.0-alpha.33)
+
+_July 1, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 💅 Allow to copy the selected rows to the clipboard (#1929) @m4theushw
+- 🚀 `Select all` checkbox click should select only filtered rows after filter (#1879) @ZeeshanTamboli
+- ⚡️ Add single select column type (#1956) @DanailH@
```

Comment:
I think that it would be great to describe more the new feature: demo/link to the docs. Without context, it's hard to understand if it's relevant for a developer or not.

Reply:
I can add a link to the docs with a small code example